src/index.js

```diff
@@ -12,11 +12,14 @@ export const renderReact = (name, component) => hypernova({
   },
 
   client() {
-    const { node, data } = load(name);
+    const payloads = load(name);
 
-    if (node) {
-      const element = React.createElement(component, data);
-      ReactDOM.render(element, node);
+    if (payloads) {
+      Array.prototype.map.call(payloads, (payload) => {
```

Comment:
Right - that `load` function that you linked returns an object, which can't be iterated over using `Array.prototype.map`